### PR TITLE
Add actions to search forms

### PR DESCRIPTION
**Pull request by zackkrida**
## Description

Adds `action="search"` to the search forms on Openverse. With this change, the user is able to submit openverse searches when JavaScript is disabled. The site is surprisingly usable without JavaScript; the core flow of searching and selecting single results works quite well.

This also fixes a rare, but occasional, bug, where the user could submit a search before the page's JavaScript bundles have finished loading and parsing. The form submits as a standard html form without an action assigned, so appears to the user like a page refresh.

## Testing Instructions

Run the frontend locally, disable JS, and submit a search from the homepage.
Additionally, do the same on `main` and observe the search does not work.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like
      `Update index.md`).
- [x] My pull request targets the _default_ branch of the repository (`main`) or
      a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [x] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible
      errors.

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: frontend/src/components/VHeader/VSearchBar/VStandaloneSearchBar.vue [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <template>
   <form
```
Comment (zackkrida) [inline frontend/src/components/VHeader/VSearchBar/VStandaloneSearchBar.vue:2]:
```suggestion
  <!-- Form action is a fallback for when JavaScript is disabled. -->
  <form
```

**Comment by zackkrida** [comment]
It would be nice to adjust this to respect the chosen locale, so it stays on the same language. Perhaps that could be a future enhancement. The number of non-js users, and non-english non-js users for Openverse is likely to be quite low.

**Comment by obulat** [comment]
> It would be nice to adjust this to respect the chosen locale, so it stays on the same language. Perhaps that could be a future enhancement. The number of non-js users, and non-english non-js users for Openverse is likely to be quite low.

I tried doing that by replacing the `action` with '../search', 'search/' or './search', but it still removes the locale :(

**Review by obulat — APPROVED** [review]
Thank you for such a quick PR :) Works well without JavaScript.
I rebased onto main to fix tests.

**Pull request merged**